Makefile

```diff
@@ -114,7 +114,7 @@ docs:
 	docker rm tiltdocs || exit 0
 	rm -fR docs/_build
 	docker build -t tilt/docs -f Dockerfile.docs .
-	docker run --name tiltdocs tilt/docs
+	docker run -e "TILT_VERSION=$$(<release_version)" -e "TILT_RELEASE_DATE=$$(<release_date)" --name tiltdocs tilt/docs
```

Comment:
this is cool, but it would be nice if it interoperated with the DATE/VERSION management that goreleaser is already doing. i'm not a big fan of having a duplicate, parallel system for managing release DATE + VERSION.